### PR TITLE
[IMP] Quality: Measure check Shop Floor

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR replaces the tablet view workflow in the Measure check doc with a new section detailing the workflow for the Shop Floor module.

### CODE REVIEW

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:154]:
```suggestion
Orders`. Select an |MO| that includes a work order for which a measure quality check is required.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:165]:
```suggestion
Process the work order's steps until the measure quality check step is reached. Click the step
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:174]: I think we'd use a guilabel when referring to fields, but since you changed it to italics, maybe you wanted to talk about tolerance as a concept? If that's the case, I think it would still make sense by removing "tolerance section" since that sounds like you're referring to something on the UI.
```suggestion
If the measurement entered is within the range specified in the *tolerance* of the |QCP|,
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:181]:
```suggestion
instructions entered in the :guilabel:`Message If Failure` tab of the |QCP|. At the bottom of the pop-up, two
```
Since you're directly referring to the UI

**Review by Felicious — APPROVED** [review]
Wonderful job updating this doc, @hojo-odoo !! I just had some really minor formatting comments, and feel free to send this over to the next stage after you take a look!

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:154]:
```suggestion
*Measure* quality checks configured for work orders **must** be completed from the *Shop Floor*
module.

To do so, begin by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing
Orders`. Select an |MO| that includes a work order for which a *Measure* quality check is required.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:159]:
```suggestion
On the |MO|, select the :guilabel:`Work Orders` tab, and click the :guilabel:`Open Work Order
(square with arrow coming out of it)` button on the line of the work order to be processed. On the
resulting :guilabel:`Work Orders` pop-up window, click the :guilabel:`Open Shop Floor` button to
open the *Shop Floor* module.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:163]:
```suggestion
center where the order is configured to be processed, and isolates the work order's card, so no
other cards are shown.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:176]:
```suggestion
|QCP|, the quality check passes, and the pop-up window moves on to the next step of the work order.
However, if the measurement entered is outside of the specified range, a new pop-up window appears,
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:199]:
```suggestion
If a quality alert must be created, exit the pop-up window by clicking the :guilabel:`✖️ (X)`
button in the top-right corner. Then, click the :guilabel:`⋮ (three vertical dots)` button on the
bottom-right corner of the work order card to open the :guilabel:`What do you want to do?` pop-up
window.
```

**Review by ksc-odoo — APPROVED** [review]
@hojo-odoo Just finished my Peer Review of this doc. Awesome job. Only a few, super minor things that need your attention. But, once you address those, it should be good to move to the next stage. 👍

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:154]:
```suggestion
*Measure* quality checks configured for work orders **must** be completed from the *Shop Floor*
module. To do so, begin by navigating to :menuselection:`Manufacturing --> Operations -->
Manufacturing Orders`. Then, select an |MO| that includes a work order for which a *Measure* quality
check is required.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:163]:
```suggestion
center where the order is configured to be processed, and isolates the work order's card, so no
other cards are shown.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:182]:
```suggestion
pop-up window, two buttons appear: :guilabel:`Correct Measure` and :guilabel:`Confirm Measure`.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:204]:
```suggestion
window.

For a complete guide on how to fill out quality alert forms, view the documentation on :doc:`quality
alerts <../quality_management/quality_alerts>`.
```

**Review by ksc-odoo — CHANGES_REQUESTED** [review]
Hey @hojo-odoo Just finished Final Reviewing this one. Another awesome, super thorough job. Just a few small modifications need your attention. Once you make those changes, tag me again, I'll give it another quick pass, and then it'll be ready to move on. Thanks!

**Comment by hojo-odoo** [comment]
@samueljlieber This is ready for technical review.

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:192]:
```suggestion
measure in the :guilabel:`Measure` field, then click :guilabel:`Validate` to complete the check and
close the pop-up window.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (ksc-odoo) [reply content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:205]: I think the spacing here can be helpful.

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (ksc-odoo) [reply content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:200]: a little separation always makes a doc a bit easier for readers to digest. And if you don't feel like the emoji use is necessary, you don't have to implement it. Just figured it would look a bit nicer. That's up to you. 👍

**Review by ksc-odoo — CHANGES_REQUESTED** [review]
hey @hojo-odoo this doc is *just about ready* for the next stage. just have a couple minor things for you to adjust. once those changes are implemented, I'll approve and move it to the next stage. thanks for all your hard work! 👍

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (hojo-odoo) [reply content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:192]: I've been told to include the "and" in cases like this.

**Review by ksc-odoo — APPROVED** [review]
I approve! Nicely done, @hojo-odoo -- this one is ready for tech review @samueljlieber

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:199]: Please add the icon descriptor 🙂
```suggestion
If a quality alert must be created, exit the pop-up window by clicking the :guilabel:`X (close)`
button in the top-right corner.
```

**Review by samueljlieber — APPROVED** [review]
Hi @hojo-odoo, this PR also looks great– nice work! I have the same change as in the `picture_check.rst` doc that I will push up after this review. Approving now!

**Comment by samueljlieber** [comment]
@StraubCreative this PR is good to go 👍

File: content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst:158]:
```suggestion
(external link icon)` button on the line of the work order to be processed. On the
```

**Review by StraubCreative — APPROVED** [review]
looks good @hojo-odoo 
same comment regarding the icon name 😄 

@robodoo delegate=hojo-odoo

**Comment by hojo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
